test/pieces-pages.js

```diff
@@ -50,6 +50,15 @@ describe('Pieces Pages', function() {
         }
       },
       afterInit: function(callback) {
+        console.log('>> afterInit');
+        // In tests this will be the name of the test file,
+        // so override that in order to get apostrophe to
+        // listen normally and not try to run a task. -Tom
+        apos.argv._ = [];
+        return callback(null);
+      },
+      afterListen: function(err) {
```

Comment:
Same function with the same name, right below (line 68)?

Reply:
Oops, fixed (caused no problems because last one wins, but still).